src/constants/languages.js

```diff
@@ -109,24 +105,24 @@ const languages = [
   { value: 'tt', label: 'Tatar' },
   { value: 'te', label: 'Telugu' },
   { value: 'th', label: 'Thai' },
-  { value: 'bo-cn', label: 'Tibetan (PRC)' },
+  { value: 'bo', label: 'Tibetan' },
   { value: 've', label: 'Tshivenḓa' },
   { value: 've', label: 'Venḓa' },
   { value: 'tr', label: 'Turkish' },
-  { value: 'tk-tm', label: 'Turkmen (Turkmenistan)' },
-  { value: 'ug-cn', label: 'Uighur (PRC)' },
+  { value: 'tk', label: 'Turkmen' },
+  { value: 'ug', label: 'Uighur' },
   { value: 'uk', label: 'Ukrainian' },
-  { value: 'wen-de', label: 'Upper Sorbian (Germany)' },
+  { value: 'wen', label: 'Upper Sorbian' },
   { value: 'ur', label: 'Urdu' },
   { value: 'uz', label: 'Uzbek' },
   { value: 'vi', label: 'Vietlabelse' },
   { value: 'cy', label: 'Welsh' },
-  { value: 'wo-sn', label: 'Wolof (Senegal)' },
+  { value: 'wo', label: 'Wolof' },
   { value: 'ts', label: 'Xitsonga'},
   { value: 'ts', label: 'Tsonga'},
```

Comment:
Fascinating, there are two possible labels for the Tsonga language. Both are valid, though I'm noting that PFE (as of PR [7134](https://github.com/zooniverse/Panoptes-Front-End/pull/7134/files)) only uses the 'Xitsonga' label

https://en.wikipedia.org/wiki/Tsonga_language

No changes necessary on either this PR or the PF PR, but worth a follow up discussion.

Reply:
Duplicate entries allow for more flexible language name search. As implemented in https://github.com/zooniverse/pandora/pull/666, these multiple entries allow users to match via endonym and exonym language names.

For additional related discussion, see: https://github.com/zooniverse/Panoptes-Front-End/pull/6729#issuecomment-1643938839, https://github.com/zooniverse/Panoptes-Front-End/pull/6732#issuecomment-1645396652

Note: this multiple label flexibility is a Pandora-only feature, so a single value/label combination must be chosen for the PFE language list. To choose single labels, we defer to the option used or preferred by local speakers.